tutorial_code/example1_2_fm_loop_correlations.py

```diff
@@ -263,8 +309,7 @@ def main():
         "all_alpha_Phi": [],
         "all_alpha_J": [],
     }
-
-    run_experiment(param, shim, stats, embeddings, 1e-5, 5e-3)
+    run_experiment(param, shim, stats, embeddings, 1e-5, 5e-3, use_cache)
```

Comment:
Would these learning rates be better defined above or as constants at the top of the file?